sdk/search/azure-search/src/main/java/com/azure/search/models/OcrSkill.java

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+package com.azure.search.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+
+/**
+ * A skill that extracts text from image files.
+ */
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "@odata.type")
+@JsonTypeName("#Microsoft.Skills.Vision.OcrSkill")
+@Fluent
+public final class OcrSkill extends Skill {
```

Comment:
LineEnding parameter is missing.

Reply:
@careyjmac I don't see it in any of our Swagger specs. When was it introduced?

Reply:
Not sure, its definitely in the documentation though: https://docs.microsoft.com/en-us/azure/search/cognitive-search-skill-ocr not a super blocker but something we should put on our backlog probably

Reply:
@Careyjmac Not probably, definitely. And it should be done soon. :)

This is not blocking for this PR though.